test/global.test.js

```diff
@@ -262,7 +320,6 @@ test('It should access the correct options property', async (t) => {
     enableCSPNonces: true,
     contentSecurityPolicy: {
       directives: {
-        ...helmet.contentSecurityPolicy.getDefaultDirectives(),
```

Comment:
With the exception of the test on L135 that explicitly tests this `getDefaultDirectives()` export, it seemed more appropriate in these tests to NOT explicitly provide these headers and instead rely on helmet's default behavior of injecting these default directives itself.